src/parser/jobs/war/modules/OGCDDowntime.ts

```diff
@@ -19,16 +19,19 @@ export class OGCDDowntime extends CooldownDowntime {
 
 		{
 			cooldowns: [this.data.actions.INNER_RELEASE],
-			// IR can sit for up to 4 GCDs, enough to reapply Eye and avoid overcapping gauge
-			allowedAverageDowntime: 10000,
+			// IR can sit for up to 5 GCDs, enough to reapply Eye and avoid overcapping gauge
+			allowedAverageDowntime: 12500,
```

Comment:
It's to allow a 1-2-3 combo and one FC. Not sure why I bumped this higher than that or the comment changed to 5, prob a mistake while thinking about PRR which isn't relevant to average downtime. Does 9000 sound alright to deal with weaving latency and 4 GCDs?

Reply:
I just realised I've already fixed this but forgot to commit it lmao.

Reply:
If it "should" be 0, but could be up to 4, I would set 5000 -- (0 * 2500 + 4 * 2500)/2.  9000ms average allowed downtime would basically imply that every time IR comes up you expect to need 3 GCDs of delay and maybe 4.  

Also, since IR changed in late 5.x to extend Eye on application, and IR is still doing that for Tempest, do we even need to allow for reapplying Tempest?  We'll already be dinging them if they let Tempest drop, does it really make sense to allow grace on the IR cooldown?